src/Common/src/Interop/Unix/System.Native/Interop.PathConf.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Sys
+    {
+        internal static int DEFAULT_PC_NAME_MAX = 255;
```

Comment:
DEFAULT_PC_PATH_MAX was removed and put behind GetMaximumPath.  The same thing doesn't need to be done for DEFAULT_PC_NAME_MAX?